addons/website_forum/controllers/website_forum.py

```diff
@@ -85,41 +85,55 @@ def _get_forum_port_search_options(self, forum=None, tag=None, filters=None, my=
             'tag': str(tag.id) if tag else None,
             'filters': filters,
             'my': my,
+            'author_id': author_id,
+            'include_answers': include_answers,
         }
 
-    @http.route(['/forum/<model("forum.forum"):forum>',
+    @http.route(['/forum/all',
+                 '/forum/all/page/<int:page>',
+                 '/forum/<model("forum.forum"):forum>',
                  '/forum/<model("forum.forum"):forum>/page/<int:page>',
                  '''/forum/<model("forum.forum"):forum>/tag/<model("forum.tag"):tag>/questions''',
                  '''/forum/<model("forum.forum"):forum>/tag/<model("forum.tag"):tag>/questions/page/<int:page>''',
                  ], type='http', auth="public", website=True, sitemap=sitemap_forum)
-    def questions(self, forum, tag=None, page=1, filters='all', my=None, sorting=None, search='', **post):
+    def questions(self, forum=None, tag=None, page=1, filters='all', my=None, sorting=None, search='', author_id=False, include_answers=False, **post):
         Post = request.env['forum.post']
 
+        author = request.env['res.users'].browse(int(author_id)).exists()
```

Comment:
Is the exists necessary ? As it adds a query ... and if you give a wrong ID, well, is it that bad ?